src/TConsole.cpp

```diff
@@ -626,7 +630,7 @@ void TConsole::resizeEvent(QResizeEvent* event)
     }
     mpMainDisplay->move(mMainFrameLeftWidth, mMainFrameTopHeight);
 
-    if (mIsSubConsole || mIsDebugConsole) {
+    if (mType & (CentralDebugConsole|ErrorConsole|SubConsole|UserWindow)) {
```

Comment:
I think the logic needs to be inverted so that the former `else` code is then *only* used for the `MainConsole` case...